project-api-tests/src/test/resources/bal-tool/v1.1.0/disttestpackage/Ballerina.toml

```diff
@@ -2,4 +2,3 @@
 org = "bctestorg"
 name = "disttestpackage"
 version = "1.1.0"
-readme = "Package.md"
```

Comment:
Let's keep this in one test case so that this scenario is also tested.

Reply:
Fixed with 0148893e661f7aeef054f422d8a2ccbf54898d71